recipes/autogen/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "5.18.7" %}
+
+package:
+  name: autogen
+  version: {{ version }}
+
+source:
+  fn: autogen-5.18.7.tar.xz
+  url: http://ftpmirror.gnu.org/autogen/autogen-5.18.7.tar.gz
+  sha1: 83dc95438470447b13f9b958871d7c3b7a868733
+
+build:
+  number: 0
+  skip: True # [win]
+
+requirements:
+  build:
+    - guile # [not win]
+    - pkg-config # [not win]
```

Comment:
Also, if you keep these, can we indent the selectors so they have the following.

* Two spaces after package name
* All the selectors are indented to be inline.